src/castra/core.cljs

```diff
@@ -87,7 +86,7 @@
                      "X-Castra-Tunnel"        "transit"
                      "X-Castra-Validate-Only" (str (boolean *validate-only*))
                      "Accept"                 "application/transit+json"}
-                    (merge (if (cell? additional-headers)
+                    (merge (if (instance? javeline.core.Cell additional-headers)
```

Comment:
Possible typo here: 
```suggestion
                    (merge (if (instance? javelin.core.Cell additional-headers)
```

What would be the difference between referring only `cell?` (i.e. `[javelin.core :refer [cell?]]`) and calling `javelin.core.Cell` directly?